src/lib.rs

```diff
@@ -555,6 +555,15 @@ impl Builder {
         self
     }
 
+    /// Add arguments to be passed straight through to clang.
+    pub fn clang_args<I>(mut self, iter: I) -> Builder
+    where I: IntoIterator, I::Item: AsRef<str> {
+        for arg in iter {
+            self.options.clang_args.push(arg.as_ref().into());
```

Comment:
perhaps:

```rust
self = self.clang_arg(arg.as_ref());
```

(not sure if you can assing to self, but perhaps using an intermediate variable is an option?)

That way if we add more stuff to `clang_arg` we can just keep this. Not a big deal if it's annoying for some reason.